src/components/ebay-radio/index.js

```diff
@@ -10,5 +10,15 @@ module.exports = require('marko-widgets').defineComponent({
                 value: target.value
             });
         }
-    }
+    },
+    handleFocus: forwardEvent('focus')
 });
+
+function forwardEvent(eventName) {
+    return function(originalEvent, el) {
+        this.emit(`radio-${eventName}`, {
```

Comment:
@ianmcburnie Should we have a check for disabled as we do in the handleClick event too?
Also same as above, I'd combine the click and focus and use forwardEvents.

Reply:
I don't think a disabled radio can gain keyboard focus, so we probably wont run into that.